raspi-blinka.py

```diff
@@ -12,13 +12,28 @@
 shell = Shell()
 shell.group="Blinka"
 default_python = 3
+blinka_minimum_python_version = 3.6
 
 def default_python_version(numeric=True):
     version = shell.run_command("python -c 'import platform; print(platform.python_version())'", suppress_message=True, return_output=True)
     if numeric:
         return float(version[0:version.rfind(".")])
     return version
 
+def get_python3_version(numeric=True):
+    version = shell.run_command("python3 -c 'import platform; print(platform.python_version())'", suppress_message=True, return_output=True)
```

Comment:
do you need to call the shell? we're in python already

Reply:
Yeah, just in case sudo apt upgrade installed anything new.

Reply:
kk!